spec/requests/api/entrypoint_spec.rb

```diff
@@ -36,4 +36,15 @@
     collection_names = response.parsed_body['collections'].map { |c| c['name'] }
     expect(collection_names).to eq(collection_names.sort)
   end
+
+  it "returns server_info" do
+    api_basic_authorize
+
+    run_get entrypoint_url
+
+    expect(response.parsed_body['server_info']).to be_kind_of(Hash)
+    expect(response.parsed_body['server_info']['version']).to eq(Vmdb::Appliance.VERSION)
+    expect(response.parsed_body['server_info']['build']).to eq(Vmdb::Appliance.BUILD)
+    expect(response.parsed_body['server_info']['appliance']).to eq(MiqServer.my_server.name)
+  end
```

Comment:
@himdel, Thanks for adding the tests.  

Could you simplify 45-48 with a single expectation.

maybe something like this (+/- keeping rubocop alignment happy):

```ruby
expect(response.parsed_body).to include(
   "server_info" => a_hash_including(
      "version" => Vmdb::Appliance.VERSION,
      "build" => Vmdb::Appliance.BUILD,
      "appliance" => MiqServer.my_server.name
   )
)
```

Thanks.